scripts/autogen.py

```diff
@@ -35,8 +35,8 @@
 import generate_tf_guides
 
 
-EXAMPLES_GH_LOCATION = "keras-team/keras-io/blob/master/examples/"
-GUIDES_GH_LOCATION = "keras-team/keras-io/blob/master/guides/"
+EXAMPLES_GH_LOCATION = os.path.join("keras-team","keras-io","blob","master","examples")
```

Comment:
1. Prefer using `Path` objects (this may require some more code changes, i.e. casting to `str` when string operations are needed)
2. Make sure to follow `black` formatting conventions